sanic/http.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# >>
+#     Copyright (c) 2016, Blake VandeMerwe
```

Comment:
nah, I gotta redo and squash this pull request -- too many formatting errors and a sneaky file got in there somehow from the cookies branch :)